### PR TITLE
Fix: Bump github.com/gopcua/opcua from 0.4.0 to 0.5.2

### PR DESCRIPTION
**Pull request by cool-wangtongzhou**
fix: Bump github.com/gopcua/opcua from 0.4.0 to 0.5.2

### CODE REVIEW

**Comment by leepala** [comment]
need to merge after fix sqlserver flaky test

File: examples/httpDeviceShifu/deployment/http-deviceshifu-configmap.yaml [hunk 1]
```diff
@@ -30,3 +36,5 @@ data:
           instruction: status
           pushSettings:
             telemetryCollectionService: push-endpoint-2
+
+
```
Comment (tomqin93) [inline examples/httpDeviceShifu/deployment/http-deviceshifu-configmap.yaml:40]: remove empty lines

File: examples/httpDeviceShifu/deployment/http-edgedevice.yaml [hunk 2]
```diff
@@ -8,3 +8,5 @@ spec:
   connection: Ethernet
   address: 192.168.14.163:12345
   protocol: HTTP
+
+
```
Comment (tomqin93) [inline examples/httpDeviceShifu/deployment/http-edgedevice.yaml:12]: remove empty lines

File: examples/httpDeviceShifu/deployment/http-deviceshifu-configmap.yaml [hunk 3]
```diff
@@ -13,6 +13,12 @@ data:
     instructions:
       status:
       info:
+      movefl:
+      movefr:
+      movef:
+      movebr:
+      movebl:
+      moveb:
```
Comment (tomqin93) [inline examples/httpDeviceShifu/deployment/http-deviceshifu-configmap.yaml:21]: where are these instructions coming from?

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/driverProperties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+driverSku: Edgenesis Mock Device
+driverImage: edgenesis/mockdevice:v0.0.1
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/driverProperties:2]: what are these?

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/instructions [hunk 5]
```diff
@@ -0,0 +1,2 @@
+instructions:
+  cmd:
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/instructions:2]: ditto

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/telemetries [hunk 6]
```diff
@@ -0,0 +1,2 @@
+telemetrySettings:
+  telemetryUpdateIntervalInMilliseconds: 1000
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/telemetries:2]: ditto

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@cool-wangtongzhou added some comment, please take a look

**Comment by cool-wangtongzhou** [comment]
> @cool-wangtongzhou added some comment, please take a look
I have deleted unnecessary things, you can take a look.

**Review by tomqin93 — APPROVED** [review]
LGTM

File: pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go [hunk 7]
```diff
@@ -166,7 +166,7 @@ func establishOPCUAConnection(ctx context.Context, address string, setting *v1al
 	}
 
 	options = append(options, opcua.SecurityFromEndpoint(ep, ua.UserTokenTypeFromString(*setting.AuthenticationMode)))
-	opcuaClient := opcua.NewClient(address, options...)
+	opcuaClient, err := opcua.NewClient(address, options...)
```
Comment (leepala) [inline pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go:169]: need to check error

**Pull request merged**